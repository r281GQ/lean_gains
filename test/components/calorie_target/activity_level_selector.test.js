import React from 'react';
import { shallow } from 'enzyme';
import ActivityLevelSelector from './../../../src/components/calorie_target/activity_level_selector';

describe('ActivityLevelSelector test', () => {
  it('should render', () => {
    const component = shallow(
      <ActivityLevelSelector/>
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
