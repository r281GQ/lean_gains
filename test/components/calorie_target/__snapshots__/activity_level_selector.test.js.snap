// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityLevelSelector test should render 1`] = `
<Field
  component={[Function]}
  floatingLabelText="Select you activity level"
  fullWidth={true}
  name="activity"
>
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Sedentary"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value={1.2}
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Lightly active"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value={1.375}
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Moderately active"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value={1.55}
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Very active"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value={1.725}
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Extremely active"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value={1.9}
  />
</Field>
`;
